packages/palette/src/elements/Menu/Menu.tsx

```diff
@@ -77,10 +78,15 @@ export const MenuItem: React.FC<MenuItemProps> = ({
   py = 1,
   textWeight = "medium",
   textColor,
+  hasLighterTextColor,
   ...props
 }) => {
   return (
-    <MenuLink href={href} {...props}>
+    <MenuLink
+      href={href}
+      {...props}
+      hasDarkerBacgroundColor={hasLighterTextColor}
```

Comment:
#superminor I think it would be slightly clearer/simpler to keep the prop naming here (so `hasLighterTextColor={hasLighterTextColor}`). But that's not a blocker!

There is a little typo (`Bacground`) 🙈. Please at least fix that and/or make the naming change, and then merge away. 😄 Thanks for bearing with me on this!!

Reply:
Haha oops! for sure!